pkg/common/utils.go

```diff
@@ -81,3 +81,20 @@ func GetAccountNumFromAWSID(AWSID string) (string, error) {
 	accountNum := (z & mask) >> 7
 	return fmt.Sprintf("%012d", accountNum), nil
 }
+
+// SliceContainsString searches a slice to determine if it contains a specified string.
+func SliceContainsString(origTargetString string, stringSlice []string, ignoreCase bool) (bool, string) {
```

Comment:
question: How large are you anticipating `stringSlice` being?

Reply:
I imagine most cases won't be huge, but didn't really have a limit in mind.